### PR TITLE
catalog_grid

### PR DESCRIPTION
**Pull request by maksym-kostetskyi**
https://maksym-kostetskyi.github.io/layout_catalog_grid

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,37 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  grid-template-rows: repeat (2, min-content);
+  justify-content: center;
```
Comment (loralevitska) [inline src/styles/_catalog.scss:5]:
```suggestion
  justify-content: center;
  grid-template-columns: min-content;
```

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -1,22 +1,37 @@
...
+
+  @media (max-width: 487px) {
+    grid-template-columns: min-content;
+  }
```
Comment (loralevitska) [inline src/styles/_catalog.scss:9]:
```suggestion
```

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,22 +1,37 @@
...
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    grid-template-columns: repeat(2, min-content);
+  }
```
Comment (loralevitska) [inline src/styles/_catalog.scss:13]:
```suggestion
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, min-content);
  }
```

File: src/styles/_catalog.scss [hunk 4]
```diff
@@ -1,22 +1,37 @@
...
+
+  @media (min-width: 768px) and (max-width: 1023px) {
```
Comment (loralevitska) [inline src/styles/_catalog.scss:15]:
```suggestion
  @media (min-width: 768px) {
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
almost perfect!

File: src/index.html [hunk 5]
```diff
@@ -73,6 +74,10 @@
       <div class="catalog__card">
         Product card
       </div>
+      <div class="catalog__card">
```
Comment (ivanchuk-yurii) [inline src/index.html:77]: why did you add another card?)

File: src/styles/_catalog.scss [hunk 6]
```diff
@@ -1,22 +1,34 @@
...
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, min-content);
+  }
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
+    display: grid;
```
Comment (ivanchuk-yurii) [inline src/styles/_catalog.scss:24]: you should change only `.catalog`, not other elements)

File: src/styles/_catalog.scss [hunk 7]
```diff
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  grid-template-rows: repeat (2, min-content);
```
Comment (ivanchuk-yurii) [inline src/styles/_catalog.scss:4]: it doesn't work, remove this

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Review by AlisaVlasova — APPROVED** [review]
